pkg/hubble/parser/options/options.go

```diff
@@ -61,6 +62,18 @@ func Redact(logger *slog.Logger, httpQuery, httpUserInfo, kafkaApiKey bool, allo
 	}
 }
 
+// EnableL3L4PolicyEnrichment configuInfo("configured Hubble with L3/L4 network policy enrichment")res the Network Policy enrichment of Hubble Flows.
```

Comment:
I don't understand this comment, is this a typo?

Reply:
Typo. Fixed :)